javascript-loops/main.js

```diff
@@ -0,0 +1,64 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  for (currentNumber; currentNumber < 11; currentNumber++) {
+    numbers.push(currentNumber);
+  }
+  return (numbers);
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  for (currentNumber; currentNumber < 21; currentNumber += 2) {
+    evenNumbers.push(currentNumber);
+  }
+  return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  var count = 1;
+  var repeated = '';
+  for (count; count < times; count++) {
+    repeated += word;
+  }
+  return repeated;
+}
```

Comment:
Please use `while` loops for the first three:

---

![image](https://user-images.githubusercontent.com/16143771/145314656-88796df6-b026-4949-9a53-fc5e802c7fdb.png)